### PR TITLE
Add several tenure-track faculties in Indiana University Bloomington

### PR DESCRIPTION
**Pull request by gzxultra**
We have a big School of Informatics, Computing and Engineering, faculties may from different departments but meet the inclusion criteria.

Following faculties are added or updated.  👇

* Xiaojing Liao
Assistant Professor of Computer Science
* Andrew Lukefahr
Assistant Professor of Computer Science
* Filippo Menczer
Professor of Informatics and Computer Science

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

**Comment by fycus-tree** [comment]
How's that? 

The [CS PhD handbook](https://www.sice.indiana.edu/doc/graduate/F18-CS-PHD-Student-Handbook-2018-08-15.pdf) suggests that getting non-CS faculty requires individual Dean-level approval. The policy for CS PhD students seems to be

>  Identify a faculty member from CS that will serve as chair of the Advisory Committee

**Comment by gzxultra** [comment]
@fycus-tree 

Thanks for the review.

> Typically, the Chair and the two additional faculty members are from Computer Science, but other
faculty from the School of Informatics, Computing, and Engineering are also eligible with the approval of the DGS and University Graduate School.

Actually some of the faculty I listed above, like [Minje Kim](http://minjekim.com/teaching.html), have already advised on CS Ph.D. students.

**Comment by fycus-tree** [comment]
@gzxultra 
Sure, but that requires a specific waiver from both the Director of Graduate Studies & the "University Graduate School", with no real information about who in the Graduate School has the authority to provide such a waiver.

This seems like CS advising is the exception, not the norm. Students are not allowed to be advised by CS faculty unless it get specific approval.

**Comment by gzxultra** [comment]
@fycus-tree 

[The Inclusion criteria](http://csrankings.org/faq.html) is: 
> What are the criteria for including faculty?
The criteria for inclusion are that anyone who is a full-time, tenure-track faculty member on a given campus who can solely advise PhD students in Computer Science can be included in the database. This approach thus extends the reach of the database to a number of faculty from other departments who have adjunct appointments with a CS department or similar that let them advise CS PhD students.


[This issue](https://github.com/emeryberger/CSrankings/issues/480) points out the inclusion criteria is about the research advising, not the approval things(which is actually simple than it looks like).

> For non-CS faculty advising CS PhD students, a link to at least one CS thesis showing that this person was an advisor.

These professors do research on cs topics, publish papers on cs conferences, advised on cs students, some of them even have an adjunct title from cs department. I did not include faculties from other disciplines.

cc @emeryberger

**Comment by emeryberger** [comment]
No, the current inclusion criteria is exactly as described by @fycus-tree. That means these faculty are ineligible.

**Comment by gzxultra** [comment]
@emeryberger  @fycus-tree 
I have adjusted and rebased my PR, the left professors should perfectly fit into the current criteria.

File: csrankings.csv [hunk 1]
```diff
@@ -13611,3 +13611,6 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
 Özgür Simsek,University of Bath,https://www.mpib-berlin.mpg.de/en/staff/oezguer-simsek,z1BYZG0AAAAJ
 Öznur Özkasap,Koç University,http://home.ku.edu.tr/~oozkasap,v2WiJeIAAAAJ
 Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
+Andrew Lukefahr,Indiana University,http://homes.sice.indiana.edu/lukefahr/,tZ4fdX0AAAAJ
+Andrew Lumsdaine,Indiana University,https://www.sice.indiana.edu/faculty-research/directory/profile.html?profile_id=246,rc_EJ2kAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Andrew Lumsdaine is currently on a non-sabbatical leave to a national lab, so he does not meet the inclusion criteria.

**Review by emeryberger — CHANGES_REQUESTED** [review]
Please remove the entry for Lumsdaine and I will merge.

File: csrankings.csv (hunk 1 above)
Comment (gzxultra) [reply csrankings.csv]: done!

**Review by emeryberger — APPROVED** [review]

**Comment by emeryberger** [comment]
Thanks!

**Pull request merged**